pkg/cloudprovider/providers/gce/gce_disks_test.go

```diff
@@ -37,7 +38,8 @@ func TestCreateDisk_Basic(t *testing.T) {
 	/* Arrange */
 	gceProjectId := "test-project"
 	gceRegion := "fake-region"
-	fakeManager := newFakeManager(gceProjectId, gceRegion)
+	zonesWithNodes := []string{"zone1"}
```

Comment:
Can we remove the managedZones setting in these tests?

Reply:
many of the tests rely on "managed zones", especially the ones that delete instances. I can remove them on the ones that don't require doing a delete if you think that would be useful

Reply:
nah, I'm just trying to see if there's a way we can avoid duplicate definitions.  Maybe can you change the managedZones to use zonesWithNodes?